.pre-commit-config.yaml

```diff
@@ -64,3 +64,28 @@ repos:
         exclude: >-
           ^docs/changelog/(\d+\.(bugfix|feature|deprecation|breaking|doc|misc).rst|README.rst|template.jinja2)
         files: ^docs/changelog/
+      - id: changelogs-user-role
+        name: changelog files have a non-broken :user:`name` role
+        language: system
+        entry: >-
+          sh -c '
```

Comment:
Oh, we can't accept any Linux checks. Use python to make it cross platform.

Reply:
I'm trying to figure out if I can come up with a pygrep check but can't think of a solution. `--negate` isn't helpful. Any ideas?

Reply:
maybe this?

```
args: [--multiline, --negate]
entry: :user:`[^`]+`
```

Reply:
Alright... I think I came up with something working.

Reply:
> maybe this?

No, this matches files not having `:user:` and we agreed not to enforce this. FWIW I've committed the solution that works with different tests I came up with.